licenses/SSPLv1.txt

```diff
@@ -496,7 +496,7 @@
   application program interfaces, automation software, monitoring software,
   backup software, storage software and hosting software, all such that a
   user could run an instance of the service using the Service Source Code
-  you make available.  
+  you make available.
```

Comment:
Ha! Tell Mongo to fix it on their repo (:

Reply:
hahahaha pre-commit things 🦤